pkg/reportingplugins/mercury/aggregate_functions.go

```diff
@@ -47,7 +47,7 @@ func GetConsensusAsk(paos []ParsedAttributedObservation) *big.Int {
 
 // GetConsensusCurrentBlock gets the most common (mode) block hash/number.
 // In the event of a tie, use the lowest numerical value
-func GetConsensusCurrentBlock(paos []ParsedAttributedObservation, f int) (hash []byte, num int64, err error) {
+func GetConsensusCurrentBlock(paos []ParsedAttributedObservation, f int) (hash []byte, num int64, ts uint64, err error) {
```

Comment:
@samsondav Is it possible that consensus block num and consensus hash belong to different blocks? If not, please consider this update. 
I commented out the related tests for now, depending on the review here I will either fix them or bring all back.

Reply:
Ok. Does it have to be done the same way for the timestamp as well? 
Alternatively, the timestamp can be taken from the observation chosen by hash or by block number.

Reply:
And I'd appreciate if you could explain why block number can be different for the same block hash.

Reply:
Yes it should be done the same way for the timestamp. It could potentially be different e.g. on chains with high probability of re-orgs where you can have two nodes that observe a block with the same number but different hash.